### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by WW3** [LunarLogic/starability#8]
Screen reader does not read the label title which is highly informative. Consider adding aria-label="[title text] + [label text]". For example aria-label="Average 3 stars". FYI, aria-label overrides the label text itself so screen reader will only hear this once.

**Comment by szynszyliszys** [comment]
@WW3 thank you for the comment. Is it true for all screen readers? If so, I will try to find a moment to fix this.

**Comment by WW3** [comment]
@szynszyliszys guess so though i haven't tested it besides WIN/NVDA. see https://www.w3.org/TR/accname-aam-1.1/#mapping_additional_nd_te

**Comment by szynszyliszys** [comment]
@WW3 sounds good enough, added aria-label attribute. Thank you for your help. :)

**Comment by szynszyliszys** [comment]
@WW3 just did testing with Voice Over. It doesn't read any label in the new Chrome (56), when the aria-label is in there. It turns out that there can be [either label text or aria-label attribute](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute). Apparently the text in the label is considered visible here. I will stick to the label text then and remove the aria-label attribute.
